### DISCUSSION TITLE
How to add the testcases in release note template

### DISCUSSION THREAD

**Discussion by ragul5194** [rfennell/AzurePipelines#1186] [General]
Hi I m @ragul5194,
We will generally attach the task id with the PR. So we used your release note template and took the parent only script ,its works good. but the testcases like tests and tested by will be mapped with parent(bug,pbl).
so it is possible to get the  tests or tested by data which is associated with parent in release notes.

**Comment by rfennell** [comment]
The way the manual tests runs (and hence manual tests) are retrieved is by their association with the build run. 

```node
    agentApi.logDebug(`Get batch of manual test runs [${runSkip}] - [${runSkip + batchSize}]`);
                var runs = await (testAPI.getTestRuns(
                    teamProject,
                    `vstfs:///Build/Build/${buildid}`,
                    null, // owner
                    null, // tmpiRunId
                    null, // planId
                    true, // include details
                    false, // shows both manual and automated
                    runSkip,
                    batchSize));
                // this returns both manual and automated we need to filter
               runs = runs.filter(run => run.isAutomated === false);
```
This means that it is independent of the PR.

So when you run the manual tests you must make sure they are associated with the correct build.

**Comment by ragul5194** [comment]
Hi ,
i think you understand in different concept.
our testers will associate their testcase workitem with Bug and PBL.so it is possible to get the testcases associated with the parent workitem.
